### DISCUSSION TITLE
中央的天氣

### DISCUSSION THREAD

**Discussion by Sherry9106** [1101-GS4719-C/Python-Community-GS4719-C#1798] [General]
中央的天常常陰晴不定，有時候今天明明出大太陽，明天就可能是刮風下雨冷得要命，忽冷忽熱的真的很難穿衣服，也很容易感冒。
最近有寒流要來了，大家注意保暖，期末考也要來了，小心不要感冒了...
https://tw.news.yahoo.com/%E5%85%A5%E5%86%AC%E9%A6%96%E6%B3%A2%E5%AF%92%E6%B5%81%EF%BC%8110%E7%B8%A3%E5%B8%82%E4%BD%8E%E6%BA%AB%E7%89%B9%E5%A0%B1-%E6%98%8E%E5%BE%8C%E5%A4%A9%E6%9C%80%E5%86%B7%E6%81%90%E4%B8%8D%E5%88%B07%E2%84%83-040327204.html

**Comment by Patsy17** [comment]
真的一下忽冷忽熱的，很難穿衣服～
不過不變的是一直都潮濕，我朋友的房間整天開除濕機還是長了菇類..

**Reply by Sherry9106** [reply]
整天開除濕機還是那麼潮濕？！我只知道我朋友外宿的地板是磁磚，他說天氣冷又陰雨綿綿的時候都會反潮，鞋子還會發霉…

**Comment by 111426027** [comment]
超級討厭中央的天氣，冬天的時候風超大又很冷，非常不適合怕冷的人居住。

**Reply by Sherry9106** [reply]
真的，但是夏天又熱到昏😵‍💫

**Comment by v0ouo** [comment]
中央的天氣真的變化超大，雖然不知道是不是氣候變遷的問題。我記得12月中那時有一天還超熱，熱到穿短袖還是會熱的那種。然後隔天忽然就變超冷，我當時被子蓋太少起來就一直打噴嚏 ，後來還得了小感冒。

**Comment by uwje** [comment]
桃園這邊的天氣真的很奇怪，有時候看明明出大太陽，結果出去的時候冷得要死，有時明明早上看起來還好好的，結果下午下大雨，真的很難處理。

**Comment by B1nn9** [comment]
今年冬天特別熱的原因應該是因為聖嬰現象發展較強烈的關係，雖然還是會有幾波冷空氣南侵，但整體而言氣溫較氣候平均值是相對偏高的趨勢，所以12月的時候才會前一天還很熱，但隔天突然又變超冷。

**Comment by jim1743** [comment]
我真的感受到北部的天氣與南部天氣的不同，在南部的時候，冬天的衣服只要過了冬天，就可以收起來了，但來到中央，天氣忽冷忽熱的，導致任何時候的衣服都要留著，以備不時之需，再來，有時候因為溫差變化太大，我就有可能因此得了個小感冒，真的很難受。

**Comment by EPHMR0524** [comment]
中央大學的天氣其實也沒有我當初想像的奇怪，可能是因為我住在天氣更奇怪的地方吧哈哈，雨下很多是真的，常常是濕冷的天氣，總是要帶傘出門，下雨天鞋子濕掉真的是很難受，下雨加上冬天的冷風真的會讓人不想要出門，要不是有上課，能不出門是一定不會走出去宿舍的。

**Comment by 110101501** [comment]
由於我是基隆人，本身就能體會濕冷的天氣。
原以為中壢比基隆南邊一些，相較之下不會比基隆冷。但事實而言，中壢這一帶的冬天均溫都比基隆冷個1-2度，這讓原本就很怕冷的我更受不了。

**Comment by YOGA012** [comment]
身為屏東人，中央夏天的天氣對我來說算蠻舒服的，但只要到冬天就跟地獄一樣，又濕又冷風又大，過了兩年依舊無法適應，希望畢業前的冬天我都能順利熬過去。

**Comment by wanka2030** [comment]
身為宜蘭人下雨天已經習以為常，但中央的天氣真的大好大壞、陰晴不定，有時候早上穿著短袖出門，要回家時便需要加上一件外套才能安全抵達。下雨天的時候更扯，雨傘的防護根本接近於零，因為大風依舊會把大雨吹進傘內，還是等到雨小一點再走好了！

**Comment by William08290829** [comment]
面對忽冷忽熱的天氣，我們可以隨時調整衣物，保持適應。在天氣轉涼時穿著較厚的衣物，並注意保暖；當天氣轉暖時可以脫下多餘的外套。同時，保持水分補充，避免長時間暴露在陽光下，並注意室內外溫差的影響，以確保身體舒適和健康。

**Comment by thomasxshen** [comment]
中央天氣真的超破，從大一就這麼覺得，時不時一直下雨真的讓人很躁， 衣服和棉被還很容易發霉，風還很大，冬天的時候感覺比台北還冷，超級恐怖的。

**Comment by Mitochondrionjessica** [comment]
中央一下雨就直接變水樂園，宿舍出來有小河，路上是水坑。我覺得買一雙雨鞋是很值得的投資。

**Comment by YuanWang08** [comment]
其實與台北相比，我覺得中央的天氣其實也還好，而且沒有那麼常下雨，有時候突然有點冷也不會太冷，但我通常還是風衣外套不離身，避免真的風大的時候著涼感冒。

**Comment by yxin214** [comment]
可能因為從小住北部我覺得自己沒有那麼不適應這裡的天氣，但這邊冬天真的比台北冷好多，之前從來沒有晚上睡覺要蓋三條被子的情況。而且雨天時風超大，雨傘都被吹開花...

**Comment by alexender0104** [comment]
身為台北人，我覺得中壢的天氣真的蠻神奇的，比起台北這裡潮濕非常多，導致體感溫度很低，風也不知道為甚麼超大，在學校內走著走著，雨傘好幾次都差點被吹走，有時常常看到身旁的人雨傘甚至被吹開花!

**Comment by OG-Matcha** [comment]
身為桃園人，我只能說這就爛桃園，天氣已經習慣了，中央特別的就是風不知道從哪邊吹來的，可能山上風比較大，逆風的時候腳踏車被定住都騎不動，下雨又風大的時候路上都是浪，在大一點可能都可以衝浪了

**Comment by happy246810** [comment]
身為台北人，我自認我應該相較其他人較禦寒，但因為學校蓋在山坡上，同時風又大，導致冬天時因為水氣極重，讓人感覺比平常更冷，而且還會感覺全身濕濕的，這種感覺我超級討厭，因為會覺得黏黏的。雖然台北也很潮濕，但根本比不上中央。

**Comment by 110101501** [comment]
身為基隆人，過去已經很能體會多變、濕冷的天氣。
但在中央，我認為可能因地理因素的影響，使得天氣來得更加多變，更需要留意自身狀況，而做出對應的措施，不要著涼了。

**Comment by hbh10601** [comment]
我懷疑你在偷唱歌，但我沒有證據(X
我也是一個超討厭下雨的人，這幾天還突然變冷不知道是怎樣，覺得又快感冒了。
不過天氣好時候的中央還是蠻美麗的，尤其是草地那邊。

**Comment by cassy0201** [comment]
我覺得中壢的天氣其實跟台北差不多，但因為走路空間都沒有騎樓或任何遮蔽物，體感上會更不方便

**Comment by YeeDaniel** [comment]
這邊先推薦Joyce的【都是weather你】，中壢真的爛天氣欸，對於住在中部的我來說，冬天真的生不如死，每天都需要穿4件以上的衣服... 梅雨季又一直下雨，超級躁

**Comment by yubo0oo** [comment]
雨下整夜 我的愛溢出就像雨水
院子落葉 跟我的思念厚厚一疊
幾句是非 也無法將我的熱情冷卻
妳出現在我詩的每一頁

雨下整夜 我的愛溢出就像雨水
窗台蝴蝶 像詩裡紛飛的美麗章節
我接著寫 把永遠愛妳寫進詩的結尾
妳是我唯一想要的了解
